lib/Controller/OutboxController.php

```diff
@@ -77,15 +77,9 @@ public function index(): JsonResponse {
 	 * @return JsonResponse
 	 */
 	public function show(int $id): JsonResponse {
-		try {
-			$message = $this->service->getMessage($id, $this->userId);
-			$this->accountService->find($this->userId, $message->getAccountId());
-			return JsonResponse::success($message);
-		} catch (ClientException $e) {
-			return JsonResponse::fail($e->getMessage(), $e->getHttpCode());
-		} catch (DoesNotExistException $e) {
-			return JsonResponse::fail(null, Http::STATUS_NOT_FOUND);
-		}
+		$message = $this->service->getMessage($id, $this->userId);
+		$this->accountService->find($this->userId, $message->getAccountId());
```

Comment:
doesn't the combined query for ID and user ID above already catch non authorized access?

Reply:
Nice, another line of code gone :D